src/bootstrap/dist.rs

```diff
@@ -313,16 +313,8 @@ pub fn rust_src_location(build: &Build) -> PathBuf {
 pub fn analysis(build: &Build, compiler: &Compiler, target: &str) {
     println!("Dist analysis");
 
-    if build.config.channel != "nightly" {
```

Comment:
I think this step should remain in the form of `if !build.config.rust_save_analysis`, right?